codalab/client/local_bundle_client.py

```diff
@@ -320,7 +322,15 @@ def kill_bundles(self, bundle_uuids):
         """
         check_bundles_have_all_permission(self.model, self._current_user(), bundle_uuids)
         for bundle_uuid in bundle_uuids:
-            self.model.add_bundle_action(bundle_uuid, BundleAction.kill())
+            if not self.launch_new_worker_system:
+                self.model.add_bundle_action(bundle_uuid, BundleAction.kill())
```

Comment:
Put a comment somewhere that once we switch over to the new worker system, all this code under launch_new_worker_system will be deleted.